src/Discord.Net.WebSocket/DiscordSocketConfig.cs

```diff
@@ -25,6 +25,17 @@ namespace Discord.WebSocket
     /// </example>
     public class DiscordSocketConfig : DiscordRestConfig
     {
+        /// <summary>
+        ///     Gets or sets the cache provider to use 
```

Comment:
```suggestion
        ///     Gets or sets the cache provider to use.
```
Just for consistency.